locations/spiders/stc_se.py

```diff
@@ -0,0 +1,21 @@
+import scrapy
+
+from locations.categories import Categories, apply_category
+from locations.dict_parser import DictParser
+
+
+# Sats (Gym chain in the nordics) is active in SE, NO, DK, FI.
+# This spider is scraping the gym centers in all countries.
+class StcSESpider(scrapy.Spider):
+    name = "stc_se"
```

Comment:
If this is giving data for multiple countries, we'd want to drop the country code from the names. Also the file name.

```suggestion
# Sats (Gym chain in the nordics) is active in SE, NO, DK, FI.
# This spider is scraping the gym centers in all countries.
class StcSpider(scrapy.Spider):
    name = "stc"
```

Reply:
ah sorry, that comment is a leftover from the other spider and should be removed...